posts/heatmaps-clustering/index.qmd

```diff
@@ -25,12 +25,20 @@ _Question:_ Is there another/better way to re-order the indices? I don't think a
 
 We will use the USA presidential speeches to walk through the clustering, and then do the plotting using altair for interactivity - as much as I like seaborn (and it's [next generation API](https://seaborn.pydata.org/nextgen/) looks very cool), interactivity is great. Then again from the src, it looks like a lot of thought has been put into `clustermap`, so there might be other reasons to use it.
 
-This is therefore a trivial post, but I often find myself wanting to do this. Indeed, I will probably refer back to it myself. The data is chosen arbitrarily, and the reordering doesn't seem to add much in this case, though one could probably have fun checking political party, etc.
+This is therefore a trivial post, but will hopefully turn out to be a useful note for when we want to do this in future. The data is chosen arbitrarily, and the reordering doesn't seem to add much in this case, though one could probably have fun checking political party, etc.
```

Comment:
Ok so I was looking at the wrong PR 🤦 But good to see you had already modified it. The point about this as a footnote maybe could still be relevant though? Although admittedly it already reads better.

Reply:
I'd probably still drop the "therefore"